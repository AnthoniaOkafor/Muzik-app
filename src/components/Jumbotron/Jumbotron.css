.jumbotron {
  display: flex;
  margin: 15px 45px;
  flex-wrap: wrap;
}

.main {
  flex-basis: 55%;
  height: 60vh;
  margin-top: 30px;
  margin-left: 20px;
  padding: 30px 40px 10px 10px;
}

.photo {
  
  background-image: url("../../../src/images/jumbotron_image.png");
  background-size: cover;
  background-position: center;
  border-radius: 50px 0px;
  box-shadow: -20px 20px #1abc9c;
  -webkit-box-shadow: -20px 20px #1abc9c;
  -moz-box-shadow: -20px 20px #1abc9c;
  flex-grow: 1;
  height: 60vh;
  opacity: 1;
  padding: 0px 20px 0px 0px;
}

@media (max-width: 740px) {
  .photo {
    order: 1;
    flex-basis: 100%;
    margin-left: 10px;
    
  }
  .main {
    order: 2;
  }

}