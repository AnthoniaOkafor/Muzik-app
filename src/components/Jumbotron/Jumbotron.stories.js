import React from "react";
import { storiesOf } from "@storybook/react";
import { withInfo } from "@storybook/addon-info";
import Jumbotron from "./index";

storiesOf("Jumbotron", module)
  .addDecorator(
    withInfo({
      text: "This is the Jumbotron Component"
    })
  )
  .add("Jumbotron", () => <Jumbotron />);
