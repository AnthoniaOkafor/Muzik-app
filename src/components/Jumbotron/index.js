import React from "react";
import "./Jumbotron.css";
import Header from "../Header/index";
import Text from "../Text/index";
// import Image from '../Image/index';
// import JumbotronImage from "../../../src/images/jumbotron_image.png";

const Jumbotron = () => {
  return (
    // <Image source={require("../images/jumbotron_image.png")} alt="Jumbotron Image" height="572px" width="572px" />
    // <Image source={JumbotronImage} alt="Jumbotron Image" height="400px" width="400px" boxshadow="greenshadow" />

    <div className="jumbotron">
      <div className="main">
        <div className="words" />
        <Header
          heading="Control the vibe at events you attend"
          color="#1ABC9C"
          size={1}
        />
        <Text
          text="Take control of the party, get a clue of the DJ playing, their playlist and vote for song you wish to hear."
          size="small"
          color="greyish"
        />
        <div className="search" />
      </div>
      <div className="photo" />
    </div>
  );
};

export default Jumbotron;
