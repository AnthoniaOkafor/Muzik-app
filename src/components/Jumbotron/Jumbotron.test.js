import React from "react"
import Jumbotron from './index'
import {mount} from 'enzyme'


describe ('Jumbotron Component render', () => {
    it('renders Jumbotron without breaking', () => {
        const wrapper = mount(<Jumbotron />);
        expect(wrapper).toMatchSnapshot();
    });

    it('renders Header with heading', () => {
        const wrapper = mount(<Jumbotron />);
        const headerHeading = wrapper.find('.heading');
        expect(headerHeading.text()).toBe('Control the vibe at events you attend');
    });
    it('renders Text with text', () => {
        const wrapper = mount(<Jumbotron />);
        const textText = wrapper.find('.greyish');
        console.log(wrapper.debug());
        expect(textText.text()).toMatch('Take control of the party');
    });

});