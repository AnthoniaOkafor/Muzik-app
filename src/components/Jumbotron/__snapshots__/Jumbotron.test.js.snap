// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jumbotron Component render renders Jumbotron without breaking 1`] = `ReactWrapper {}`;
